webapp/channels/src/actions/views/drafts.ts

```diff
@@ -153,6 +153,16 @@ export function setDraftsTourTipPreference(initializationState: Record<string, b
     };
 }
 
+export function setGlobalDraftSource(key: string, isRemote: boolean) {
```

Comment:
Would it make sense to have a 3rd function which would call the other 2? e.g:

```ts
setGlobalDraft() {
  setGlobalItem()
  setGlobalDraftSource()
}
```

0/5

Reply:
Yes, you're right. Actually, I thought about it as well at some point because we are always changing both of them either way. If we don't ever need to change just one, that's a cleaner way to go indeed.